src/Microsoft.ML.CodeGen/Templates/Console/ConsumeModel.tt

```diff
@@ -0,0 +1,46 @@
+﻿<#@ template language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ include file="Annotation.tt" #><#if(Target == CSharp.CodeGeneratorSettings.GenerateTarget.Cli){ #>
+<#CLI_Annotation();#>
+<# } else if(Target == CSharp.CodeGeneratorSettings.GenerateTarget.ModelBuilder){ #>
+<#MB_Annotation();#>
+<# } #>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.ML;
+using <#= Namespace #>.Model;
+using <#= Namespace #>.Model.DataModels;
+
+namespace <#= Namespace #>.Model
+{
+	public class ConsumeModel
```

Comment:
"Consume" is a strange choice of words for what this does.  Inference or Prediction would be a lot more descriptive.  I couldn't guess what it does until I looked up documentation.  Do you know who came up with this wording?

Reply:
@briacht

Reply:
One more related question: there's already a prediction project, why do we need a second one?

Reply:
Through user testing and reported issues we found that users struggled with the last (Code) step in Model Builder e.g. they could not figure out how to easily consume the model with the code that they were given. Additionally, the consume code became very repetitive, so we are providing users with a separate Predict method so that they only have to add namespaces, create input, and then predict.

We have used the term “consume model” in our getting started experiences / when talking about loading and using the ML model for predictions, so for now we will leave it as is for this release and then do further user testing to validate / invalidate.